### PR TITLE
An open letter about root cause

### PR DESCRIPTION
**Pull request by virginiaulrich**
This isn't so much about process, as it is about framing and remembering -- something to refer back to when we need inspiration

### CODE REVIEW

**Review by nicksnyder — APPROVED** [review]
Love it!

**Review by airamare01 — APPROVED** [review]
This is absolutely awesome stuff!

File: handbook/support/root-cause.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+Let's break it down:
+
+1. Ask yourself: How confident am I that this won't happen again; that we have done something to ensure this customer and/or other customers will not have to deal with this issue in the future? If you are not super confident, how will you reassure the customer you have done what is in their best interest and maintain their trust?
+2. Not super confident ... remind yourself and your teammates in engineering what needs to be done.
+	1. We need to get the customer into a working state as soon as possible, always
+	2. Then, we need to figure out how lasting that fix is so we can gauge how to plan for a lasting fix
+	3. File a defect or product improvement in the public tracker (the nature of the issue will determine which) and the product/engineering team can schedule the work for the lasting fix in to an upcoming iteration
+3. Share the Github link with the customer so they can follow along on progress and remind them they can ask for updates there, or in the shared Slack channel, or by emailing support@sourcegraph.com, or by talking with their CE.
```
Comment (beyang) [inline handbook/support/root-cause.md:34]:
```suggestion
3. Share the GitHub link with the customer so they can follow along on progress and remind them they can ask for updates there, or in the shared Slack channel, or by emailing support@sourcegraph.com, or by talking with their CE.
```

File: handbook/support/support-workflow.md [hunk 2]
```diff
@@ -28,21 +28,22 @@ Things happen pretty much in this order...and at every step, our decisions and a
...
-	5. If we are still working on finding the answer and the customer stops responding, we email a couple more times to see if they had a chance to look at our last response. If they still don't respond, and they have a CE, talk with them to see if they have another way to get in touch and see what's going on. If not, or ff the customer does not have a CE, AND we complete any internal tasks born from working on the ticket (like updating documentation), then it's okay to close the ticket at this point.
+	1. We help the customer figure out what they need to do and they confirm they have done it and agree the issue is resolved `AND` we complete any internal tasks born from working on the ticket (like updating documentation or filing a Github issue for a docs update).
+	2. We identify a workaround to get the customer on their way `AND` we file a defect or product improvement (in the public Github tracker) to address the underlying root cause (read [an open letter about root cause](root-cause.md) for inspiration).
```
Comment (beyang) [inline handbook/support/support-workflow.md:42]:
```suggestion
	2. We identify a workaround to get the customer on their way `AND` we file a defect or product improvement (in the public GitHub tracker) to address the underlying root cause (read [an open letter about root cause](root-cause.md) for inspiration).
```

**Review by abeatrix — APPROVED** [review]
Thanks for writing this!

**Review by DaedalusG — APPROVED** [review]
Love it 😁

**Review by christinaforney — APPROVED** [review]
✨

**Review by adeola-ak — APPROVED** [review]

File: handbook/support/root-cause.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+Your job is to make sure the customer can move forward as soon as possible `AND` that we have something set-up for the deeper work to address the underlying root cause, taking the investigation as far as you can. Product's and engineering's job (in collaboration) is to prioritize when the engineering team will do that deep work. Engineering's job (and heck yes, you can collaborate) is to do the deep work and figure out a lasting fix. We all care about our customers and there is always a lot to balance. If we each do our part, our future selves will thank us and our customers will become even bigger fans.
+
+You know this. In your bones, you know this. It's why you were hired to be a CSE. You are wonderfully innately curious. You are fiercely compassionate. Your ability to learn quickly is off the charts. Your skillset to investigate is beyond stellar. Your drive to solve problems is :chef kiss:.
+
+Sometimes, we just need to be reminded.
+
+Take another deep breathe. Have another glass of water. Open the window or step outside and feel the air on your face again. You've got this!
```
[the remark this answers is not in the document]
Comment (virginiaulrich) [reply handbook/support/root-cause.md:25]: Catching my #1 typo -- thank you! 🤦‍♀️

**Comment by virginiaulrich** [comment]
@northyg I am going to go ahead and publish this now. I know you will read it when you are able to.

**Review by northyg — COMMENTED** [review]
Looks good!

**Pull request merged**